tasks/install.yml

```diff
@@ -19,7 +19,7 @@
     name:  '{{ pihole_default_packages_apt }}'
     state: 'latest'
     update_cache: true
-  when: ansible_distribution == 'Debian'
+  when: ansible_os_family == 'Debian'
```

Comment:
In https://github.com/r-pufky/ansible_pihole/issues/17; there was comments about other packages / OS packages needing to be installed/removed for pihole to work on ubuntu.

Switching this to os_family potentially allows the role to get into a state where it 'successfully' installs but doesn't run. I haven't turned up an ubuntu box to test this yet, but is this the case here?

Reply:
In my manual test, I used Ubuntu and did not get more dependencies and works in CI PR https://github.com/juju4/ansible_pihole/actions/runs/6209020219/job/16855779572